### ISSUE TITLE
Accessibility: Adjust colours to meet at least WCAG AA

### ISSUE THREAD

**Issue by artfulrobot** [civicrm/org.civicrm.styleguide#6]
I found it hard to read text in the Shoreditch theme. My eyes are not bad and I have a properly colour managed and reasonably accurate monitor that covers the full sRGB gammut. Most users have v poor screens (less contrast) and of course many people are visually impared. CiviCRM needs to be a CRM for everyone, not just those with fantastic eyesight and fancy monitors :-)

According to http://contrastchecker.com/

None of the main colour combos are accessible at the normal font size.

![screenshot from 2017-09-11 11-05-05](https://user-images.githubusercontent.com/870343/30269588-3a444b66-96e1-11e7-9020-16181487f465.png)

I'd like CiviCRM to adhere to WCAG AAA standard, but AA standard should be a minimum, in my opinion.

Thoughts?

**Comment by artfulrobot** [comment]
PS. apols if this should have gone on the shoreditch theme itself's issue q - I can move if so.

**Comment by JoeMurray** [comment]
@colemanw @guanhuan I atrongly support these concerns. Where should issue reside?

**Comment by AkA84** [comment]
@artfulrobot thanks for reporting this, can you move it to https://github.com/civicrm/org.civicrm.shoreditch ?

**Comment by artfulrobot** [comment]
Moved to https://github.com/civicrm/org.civicrm.shoreditch/issues/56
